packages/ui-extensions/buildTargetDts.ts

```diff
@@ -81,6 +81,24 @@ export type GlobalThis = typeof globalThis & {
   shopify: Api;
 }\n`;
 
+  if (surface === 'customer-account') {
+    template = `import type {ExtensionTargets} from '../extension-targets';
+import {ExtensionNavigation} from '../api';
+type Target = ExtensionTargets[${name}];
+export type Api = Target['api'];
+export type Output = Target['output'];
+export type Navigation = ExtensionNavigation<${name}>;
+
+export type GlobalThis = typeof globalThis & {
```

Comment:
Can you remove this from the templates? It's not used because it doesn't work properly